src/main/java/com/kodilla/ecommercee/entities/UserEntity.java

```diff
@@ -0,0 +1,64 @@
+package com.kodilla.ecommercee.entities;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "USERS")
+public class UserEntity {
+	@Id
+	@GeneratedValue
+	@Column(name = "USER_ID", nullable = false)
+	private int id;
+	@Column(name = "FIRST_NAME", nullable = false)
+	private String firstName;
+	@Column(name = "LAST NAME", nullable = false)
+	private String lastName;
+	@Column(name = "ADRESS", nullable = false)
+	private String adress;
+	@Column(name = "PHONE", nullable = false)
+	private int phone;
```

Comment:
Może lepiej byłoby użyć tutaj Stringa?

Reply:
Celna uwaga, poza tym wygląda na to, że jest ok. Musimy jednak ustalić wspólne nazewnictwo bo zrobiłby się bałagan pod tym względem, jeśli wszystkie PR z encjami by przeszły.

Reply:
Podmieniłem int na String